src/pages/workspaces/workspace/analysis/modals/export-analysis-modal/export-analysis-modal.state.ts

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash/fp'
```

Comment:
I have a question about the naming convention of this file. Why aren't we calling it `useAnalysesExportState` for consistency with how we have named other files? I think this will be the first time we have a "*.state.ts" files.

Reply:
and actually, on a related note, I think we want to rename this sub-directory to ExportAnalysisModal to match the consensus forming around naming directories and files to match their main export.

Reply:
Please remember to rename this file as discussed above.

Reply:
Done!